### PR TITLE
Added rcorpora and mlbench Packages for datasets 

### LINKED ISSUE #7160: New R package rcorpora in the next release
The information in the wiki mentions either needed in dialogues or for data.  This is a package for datasets.  We discuss it quite a lot in the new help.
@Wycklife grateful if you could do the pull request as described in the wiki [here](https://github.com/africanmathsinitiative/R-Instat/wiki/Information-for-developers)

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes  #7160

This pr is not related to any dialog. It is for the two new packages for datasets

@africanmathsinitiative/developers This is ready for review.

### CODE REVIEW

File: instat/static/InstatObject/R/install_packages.R [hunk 1]
```diff
@@ -114,6 +114,8 @@ pkgs <-
     "tidytext",
     "janitor",
     "ggwordcloud",
+    "rcorpora",
+    "mlbench",
```
Comment (shadrackkibet) [inline instat/static/InstatObject/R/install_packages.R:118]: Could you add a comment for each package that explains the purpose of these packages also point to locations in the software where is used?

Examples in line `119,121` etc

**Review by shadrackkibet — CHANGES_REQUESTED** [review]

**Comment by rdstern** [comment]
mlbench includes a set of interesting data sets, designed particularly for machine learning.
rcorpora has many useful datasets, described in the R-Instat help and a forthcoming video.  They include prime numbers, Fibonacci numbers, names of dinosaurs, and the chemical elements.

**Review by shadrackkibet — APPROVED** [review]
@rdstern if you approve then we can merge.

**Review by rdstern — APPROVED** [review]
Great

**Pull request merged**